docs/csi-volume-import.md

```diff
@@ -98,3 +95,7 @@ spec:
      ports:
      - containerPort: 80
 ```
+4. Clone the imported PVC or take snapshot from it. 
```

Comment:
From my point of view, it seems we don't need to add step 4. What we need is to emphasize that step 3 is a very important step to populate the DB.

Reply:
I want users to understand the scenario and what steps to expect, I feel it is more clear to list there.